connect-examples/v2/csharp_payment/appsettings.Development.json

```diff
@@ -0,0 +1,10 @@
+﻿{

```

Comment:
Is this file needed? If so do you want to include AppSettings?

Reply:
Removed